src/test/java/uk/gov/justice/digital/delius/service/OffenderManagerService_allocatePrisonOffenderManagerTest.java

```diff
@@ -67,29 +76,25 @@ public void setup() {
                 contactService);
 
         when(offenderRepository.findByNomsNumber(any())).thenReturn(Optional.of(anOffender()));
-        when(staffService.findByOfficerCode(any())).thenReturn(Optional.of(aStaff()));
+        when(staffService.findByStaffId(anyLong())).thenReturn(Optional.of(aStaff()));
         when(probationAreaRepository.findByInstitutionByNomsCDECode(any())).thenAnswer(args -> {
-            var code = args.getArgument(0).toString();
+            final var code = args.getArgument(0).toString();
             return Optional.of(aPrisonProbationArea()
                     .toBuilder()
                     .code(code)
                     .build());
         });
-        when(prisonOffenderManagerRepository.save(any())).thenAnswer(args -> args.getArgument(0));
```

Comment:
had some fun with this test.  Under jupiter these mocks aren't lenient so tests then failed due to unnecessary mocking.  Could have made them lenient I guess, but thought better to mock out where needed instead.